several:

sorry no colors

ubuntu@ubiquityrobot:~$ roslaunch magni_demos teleop.launch
... logging to
/home/ubuntu/.ros/log/0bbf7c52-df79-11d4-a3ec-b827eb79f4ae/roslaunch-ubiquityrobot-8148.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://ubiquityrobot:40976/

SUMMARY
========

PARAMETERS
 * /joy_node/autorepeat_rate: 20
 * /joy_node/dev: /dev/input/js0
 * /rosbridge_ws/rosapi/params_glob: [*]
 * /rosbridge_ws/rosapi/services_glob: [*]
 * /rosbridge_ws/rosapi/topics_glob: [*]
 * /rosbridge_ws/rosbridge_websocket/address:
 * /rosbridge_ws/rosbridge_websocket/authenticate: False
 * /rosbridge_ws/rosbridge_websocket/delay_between_messages: 0
 * /rosbridge_ws/rosbridge_websocket/fragment_timeout: 600
 * /rosbridge_ws/rosbridge_websocket/max_message_size: None
 * /rosbridge_ws/rosbridge_websocket/params_glob: [*]
 * /rosbridge_ws/rosbridge_websocket/port: 9090
 * /rosbridge_ws/rosbridge_websocket/retry_startup_delay: 5
 * /rosbridge_ws/rosbridge_websocket/services_glob: [*]
 * /rosbridge_ws/rosbridge_websocket/topics_glob: [*]
 * /rosbridge_wss/rosapi/params_glob: [*]
 * /rosbridge_wss/rosapi/services_glob: [*]
 * /rosbridge_wss/rosapi/topics_glob: [*]
 * /rosbridge_wss/rosbridge_websocket/address:
 * /rosbridge_wss/rosbridge_websocket/authenticate: False
 * /rosbridge_wss/rosbridge_websocket/certfile: /etc/ssl/certs/ss...
 * /rosbridge_wss/rosbridge_websocket/delay_between_messages: 0
 * /rosbridge_wss/rosbridge_websocket/fragment_timeout: 600
 * /rosbridge_wss/rosbridge_websocket/keyfile: /etc/ssl/private/...
 * /rosbridge_wss/rosbridge_websocket/max_message_size: None
 * /rosbridge_wss/rosbridge_websocket/params_glob: [*]
 * /rosbridge_wss/rosbridge_websocket/port: 9443
 * /rosbridge_wss/rosbridge_websocket/retry_startup_delay: 5
 * /rosbridge_wss/rosbridge_websocket/services_glob: [*]
 * /rosbridge_wss/rosbridge_websocket/topics_glob: [*]
 * /rosdistro: kinetic
 * /rosversion: 1.12.7
 * /teleop_twist_joy/axis_angular: 0
 * /teleop_twist_joy/axis_linear: 1
 * /teleop_twist_joy/enable_button: 4
 * /teleop_twist_joy/enable_turbo_button: 6
 * /teleop_twist_joy/scale_angular: 1.0
 * /teleop_twist_joy/scale_linear: 0.3
 * /teleop_twist_joy/scale_linear_turbo: 0.75

NODES
  /rosbridge_wss/
    rosapi (rosapi/rosapi_node)
    rosbridge_websocket (rosbridge_server/rosbridge_websocket)
  /rosbridge_ws/
    rosapi (rosapi/rosapi_node)
    rosbridge_websocket (rosbridge_server/rosbridge_websocket)
  /
    joy_node (joy/joy_node)
    teleop_twist_joy (teleop_twist_joy/teleop_node)
    tf2_web_republisher (tf2_web_republisher/tf2_web_republisher)

ROS_MASTER_URI=http://localhost:11311

core service [/rosout] found
process[joy_node-1]: started with pid [8166]
process[teleop_twist_joy-2]: started with pid [8171]
process[rosbridge_ws/rosbridge_websocket-3]: started with pid [8184]
[ERROR] [1505958299.039909238]: Couldn't open joystick /dev/input/js0. Will
retry every second.
process[rosbridge_ws/rosapi-4]: started with pid [8199]
process[rosbridge_wss/rosbridge_websocket-5]: started with pid [8205]
process[rosbridge_wss/rosapi-6]: started with pid [8209]
process[tf2_web_republisher-7]: started with pid [8215]
[ WARN] [1505958299.634204521]: Shutdown request received.
[ WARN] [1505958299.634565822]: Reason given for shutdown: [new node
registered with same name]
registered capabilities (classes):
 - rosbridge_library.capabilities.call_service.CallService
 - rosbridge_library.capabilities.advertise.Advertise
 - rosbridge_library.capabilities.publish.Publish
 - rosbridge_library.capabilities.subscribe.Subscribe
 - <class 'rosbridge_library.capabilities.defragmentation.Defragment'>
 - rosbridge_library.capabilities.advertise_service.AdvertiseService
 - rosbridge_library.capabilities.service_response.ServiceResponse
 - rosbridge_library.capabilities.unadvertise_service.UnadvertiseService
registered capabilities (classes):
 - rosbridge_library.capabilities.call_service.CallService
 - rosbridge_library.capabilities.advertise.Advertise
 - rosbridge_library.capabilities.publish.Publish
 - rosbridge_library.capabilities.subscribe.Subscribe
 - <class 'rosbridge_library.capabilities.defragmentation.Defragment'>
 - rosbridge_library.capabilities.advertise_service.AdvertiseService
 - rosbridge_library.capabilities.service_response.ServiceResponse
 - rosbridge_library.capabilities.unadvertise_service.UnadvertiseService
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/rosbridge_server/rosbridge_websocket", line
205, in <module>
    application.listen(port, address, ssl_options={ "certfile": certfile,
"keyfile": keyfile})
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/tornado/web.py", line
1690, in listen
    server = HTTPServer(self, **kwargs)
  File
"/opt/ros/kinetic/lib/python2.7/dist-packages/tornado/httpserver.py", line
156, in __init__
    read_chunk_size=chunk_size)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/tornado/tcpserver.py",
line 115, in __init__
    self.ssl_options['keyfile'])
ValueError: keyfile "/etc/ssl/private/ssl-cert-snakeoil.key" does not exist
[WARN] [1505958307.873147]: Unable to start server: [Errno 98] Address
already in use Retrying in 5s.
[rosbridge_wss/rosbridge_websocket-5] process has died [pid 8205, exit code
1, cmd /opt/ros/kinetic/lib/rosbridge_server/rosbridge_websocket
__name:=rosbridge_websocket
__log:=/home/ubuntu/.ros/log/0bbf7c52-df79-11d4-a3ec-b827eb79f4ae/rosbridge_wss-rosbridge_websocket-5.log].
log file:
/home/ubuntu/.ros/log/0bbf7c52-df79-11d4-a3ec-b827eb79f4ae/rosbridge_wss-rosbridge_websocket-5*.log


On Wed, Sep 20, 2017 at 6:38 PM, Rohan Agrawal <notifications@github.com>
wrote:

> Are there any error messages in the console when your run roslaunch
> magni_demos teleop.launch ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/UbiquityRobotics/magni_robot/pull/39#issuecomment-331026482>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADxmjDaru1BUoqsIPVBgmQp7bhjEfPtJks5skb4ygaJpZM4PDSqS>
> .
>

--- a/magni_demos/launch/teleop.launch
+++ b/magni_demos/launch/teleop.launch
@@ -1,5 +1,5 @@
 <launch>
-<!--  <include file="$(find magni_bringup)/launch/base.launch" /> -->
+  <include file="$(find magni_bringup)/launch/base.launch" /> 
   <include file="$(find magni_teleop)/launch/logitech.launch" />
   <group ns="rosbridge_ws"> 
     <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">

